app/controllers/apps_controller.rb

```diff
@@ -29,7 +29,17 @@ def index
         raise ActionController::RoutingError.new('Not Found') unless app.accessible?
       end
 
-      @motd_file = MotdFile.new
+      motd_file = MotdFile.new(ENV['MOTD_PATH'])
+      case ENV['MOTD_FORMAT']
+        when 'osc'
+          @motd = MotdFormatterOsc.new(motd_file)
+        when 'markdown'
+          @motd = MotdFormatterMarkdown.new(motd_file)
+        when 'rss'
+          @motd = MotdFormatterRss.new(motd_file)
+        else
+          @motd = MotdFormatterPlaintext.new(motd_file)
+      end
```

Comment:
If `MOTD_PATH` is not set, nothing will be displayed on the dashboard, but you're right, this should probably be prevented from getting to that point.